### ISSUE TITLE
Executing summary page

### ISSUE THREAD

**Issue by rrhyne** [sourcegraph/sourcegraph#27588]
The execution summary page requires design to properly present the following:

* A representative diff
* An errors that have occurred
* Execution progress

**Comment by LawnGnome** [comment]
@rrhyne We are discussing this in sprint planning; please update which column this should be in (P0 or P1)!

**Comment by rrhyne** [comment]
@courier-new  @LawnGnome yes, this seems to be a duplicate of #31088. Designs are in Figma as noted and are awaiting feedback.

**Comment by courier-new** [comment]
Great, I'll close this one and refer to the other one in the milestone ticket!
